troposphere/static/js/components/projects/detail/resources/ProjectDetails.jsx

```diff
@@ -129,8 +129,10 @@ export default React.createClass({
                 onReportSelectedResources={this.onReportSelectedResources}
                 onRemoveSelectedResources={this.onRemoveSelectedResources}
                 previewedResource={previewedResource}
+                selectedResources={selectedResources}
                 multipleSelected={selectedResources && selectedResources.length > 1}
                 onUnselect={this.onResourceDeselected}
+                onUnselectAll={this.deselectAllResources}
```

Comment:
As noted, `onUnselectAll` got included when I realized it didn't appear I had a straightforward way to eliminate selections.